src/ViewModel/HypothesisOpener.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace eLife\Patterns\ViewModel;
+
+use eLife\Patterns\ArrayAccessFromProperties;
+use eLife\Patterns\ArrayFromProperties;
+use eLife\Patterns\SimplifyAssets;
+use eLife\Patterns\ViewModel;
+use Traversable;
+use function eLife\Patterns\mixed_accessibility_text;
+
+final class HypothesisOpener implements ViewModel
+{
+    use ArrayAccessFromProperties;
+    use ArrayFromProperties;
+    use SimplifyAssets;
+
+    private $button;
+
+    public function __construct()
+    {
+        $visibleAnnotationCount = '<span data-visible-annotation-count>&#8220;</span>';
```

Comment:
left double quotation mark? Whatever the character is, the magic number can be extracted into a constant